apps/ledger-live-desktop/package.json

```diff
@@ -186,7 +186,7 @@
     "@types/write-file-atomic": "^4.0.0",
     "@vitejs/plugin-react": "^3.1.0",
     "allure-playwright": "2.12.2",
-    "axios": "^1.3.4",
+    "axios": "^1.6.7",
```

Comment:
This update broke a test on ci recently https://ledger.slack.com/archives/C03MHGMAMRQ/p1708699888332929?thread_ts=1708689487.052639&cid=C03MHGMAMRQ

Reply:
checking 👀